projects/cobraswap/index.js

```diff
@@ -0,0 +1,40 @@
+const { request, gql } = require("graphql-request");
+const sdk = require('@defillama/sdk');
+const { toUSDTBalances } = require('../helper/balances');
+
+const graphUrl = 'https://api.thegraph.com/subgraphs/name/venomprotocol/cobra-exchange'
+const graphQuery = gql`
+query get_tvl($block: Int) {
+  uniswapFactories(
+    block: { number: $block }
+  ) {
+  		totalLiquidityUSD
+  }
+}
+`;
+
+async function tvl(timestamp) {
+  const {block} = await sdk.api.util.lookupBlock(timestamp,{
```

Comment:
This isn't really needed, it's possible to extract the block from one of the parameters to save calls. No need to wait for an iteration for this tho, I'll merge it and apply the changes myself